### DISCUSSION TITLE
Connect to existing cluster

### DISCUSSION THREAD

**Discussion by cyxer1** [Opster/opensearch-k8s-operator#840] [Q&A]
Hi, can I connect an operator to a cluster that is already in use and manage it by operator?

**Comment by swoehrl-mw** [comment]
Hi @cyxer1. This is not possible. The operator is designed to create and manage its own clusters. It cannot take over existing ones as it relies on specific structures and configuration in kubernetes resources that are just not present in clustes not created by the operator.
